api/turing/validation/validator.go

```diff
@@ -129,6 +129,43 @@ func checkTrafficRuleName(sl validator.StructLevel, fieldName string, value stri
 	}
 }
 
+func checkTrafficRuleCount(sl validator.StructLevel, fieldName string, rules models.TrafficRules) {
+	ruleLength := len(rules)
+	ruleCountDescription := "There should be either 0 or more than 1 traffic rules specified."
+	if ruleLength == 1 {
+		sl.ReportError(rules, fieldName, "trafficRule", ruleCountDescription, "")
+	}
+	// Check for presence of Default Traffic Rule
+	hasDefaultRule := false
+	noDefaultRuleDescription := "Default rule should be provided."
+	if ruleLength > 1 {
+		for _, rule := range rules {
+			if rule.Name == "default" {
+				hasDefaultRule = true
```

Comment:
Once the value is set to true, we can `break` early. But there is another condition we must check - the `default` rule is the last in the rules list (if not, it will be matched early at runtime). So, can we replace this check with:
```golang
if ruleLength > 1 {
    if rules[ruleLength-1] != "default" {
        // Report error
    }
}
```

This makes me wonder, does the name `default` really matter? Should we check for the name at all or simply that the last rule has 0 conditions ?